app/views/groups/samples/_table.html.erb

```diff
@@ -29,50 +70,56 @@
         ) %>
       </th>
     </tr>
-  </thead>
-  <tbody
-    id="group-samples-table-body"
-    class="
-      bg-white
-      divide-y
-      divide-slate-200
-      dark:bg-slate-800
-      dark:divide-slate-700
-    "
-  >
+    </thead>
+    <tbody
+      id="group-samples-table-body"
+      class="
+        bg-white
+        divide-y
+        divide-slate-200
+        dark:bg-slate-800
+        dark:divide-slate-700
+      "
+    >
     <% samples.each do |sample| %>
       <tr
         id="<%= dom_id(sample) %>"
-        class="text-sm font-normal text-slate-500 dark:text-slate-400"
+        class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
       >
-        <td class="px-4 py-3 whitespace-nowrap"><%= link_to sample.name,
-          namespace_project_sample_path(
-            id: sample.id,
-            namespace_id: sample.project.namespace.parent.full_path,
-            project_id: sample.project.path
-          ),
-          data: {
-            turbo: false
-          },
-          class: "text-grey-900 dark:text-grey-100 font-semibold hover:underline" %>
+        <td class="px-6 py-3 whitespace-nowrap"><%= link_to sample.name,
+                                                            namespace_project_sample_path(
+                                                              id: sample.id,
+                                                              namespace_id: sample.project.namespace.parent.full_path,
+                                                              project_id: sample.project.path
+                                                            ),
+                                                            data: {
+                                                              turbo: false
+                                                            },
+                                                            class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" %>
         </td>
-        <td class="px-4 py-3 whitespace-nowrap">
+        <td class="px-6 py-3 whitespace-nowrap">
           <%= viral_tooltip(title: project_path(sample.project)) do %>
             <%= link_to sample.project.abbreviated_path,
-            project_path(sample.project),
-            data: {
-              turbo: false
-            },
-            class: "text-grey-900 dark:text-grey-100 font-semibold hover:underline" %>
+                        project_path(sample.project),
+                        data: {
+                          turbo: false
+                        },
+                        class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" %>
           <% end %>
         </td>
-        <td class="px-4 py-3 whitespace-nowrap">
+        <% @fields.each do |column| %>
+          <td class="px-6 py-3 whitespace-nowrap">
+            <%= sample[column].present? ? sample[column] : sample.metadata[column] %>
```

Comment:
**nitpick (blocking):** This logic is redundant as `@fields` will only refer to `metadata`

Reply:
Updated in 2b505ec6665d25ab9f6f8725b9350582067b7efa